paddle/fluid/platform/device_context.cc

```diff
@@ -739,7 +758,7 @@ void MKLDNNDeviceContext::SetBlob(const std::string& name,
   return;
 }
 
-unsigned int MKLDNNDeviceContext::GetCachedObjectsNumber(void) {
+unsigned int MKLDNNDeviceContext::GetCachedObjectsNumber(void) const {
   unsigned int num_entries = 0;
```

Comment:
The GetCachedObjectsNumber results is num_threads * nums_input_shape ?

Reply:
Actually structure of cache is of three levels: session IDs , shapes, actual cached objects per shape. So To get quantity of  cache we iterate through sessions and their shapes. For every shape we then get amount of cached objects (for shape) and sum it up with other amounts for different shapes.